public/icons/Q.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M22.4018 22.4561C22.5152 22.4561 22.6098 22.4939 22.6854 22.5695C22.7799 22.6451 22.8271 22.7396 22.8271 22.853C22.8271 22.9665 22.7799 23.061 22.6854 23.1366C22.6098 23.2122 22.5152 23.25 22.4018 23.25H20.5021H18.3473C17.7235 23.25 17.0997 23.0515 16.4759 22.6546C15.8521 22.2576 15.3796 21.7567 15.0582 21.1518C14.0564 21.4354 13.0451 21.5771 12.0244 21.5771H11.7409C10.2854 21.5393 8.91494 21.2274 7.62957 20.6415C6.36311 20.0555 5.25732 19.2805 4.3122 18.3165C3.36707 17.3335 2.62043 16.1994 2.07226 14.914C1.52409 13.6287 1.25 12.2488 1.25 10.7744C1.25 9.3189 1.52409 7.94848 2.07226 6.66311C2.62043 5.35884 3.36707 4.22469 4.3122 3.26067C5.25732 2.29665 6.36311 1.52165 7.62957 0.935672C8.91494 0.349696 10.2854 0.0378055 11.7409 0H12.0244C13.5177 0 14.9165 0.283537 16.2207 0.850611C17.525 1.41768 18.6686 2.19268 19.6515 3.17561C20.6344 4.13964 21.4095 5.28323 21.9765 6.6064C22.5436 7.91067 22.8271 9.3 22.8271 10.7744C22.8271 12.3244 22.5152 13.7988 21.8915 15.1976C21.2677 16.5774 20.3887 17.7777 19.2546 18.7985L20.7857 22.4561H22.4018ZM22.0049 10.7744C22.0049 9.41341 21.7402 8.12805 21.211 6.91829C20.7006 5.70854 19.9918 4.65 19.0844 3.74268C18.1771 2.83537 17.1186 2.12653 15.9088 1.61616C14.6991 1.08689 13.4043 0.822257 12.0244 0.822257H11.7409C11.0982 0.841159 10.4933 0.973477 9.92622 1.21921C9.35915 1.44604 8.86768 1.75793 8.45183 2.15488C8.03597 2.53293 7.70518 2.98659 7.45945 3.51585C7.21372 4.02622 7.09085 4.57439 7.09085 5.16037V12.5607H12.0244H14.1793C14.5195 12.5607 14.8598 12.6268 15.2 12.7591C15.5591 12.8726 15.8899 13.0427 16.1924 13.2695C16.4948 13.4774 16.7689 13.7326 17.0146 14.0351C17.2604 14.3186 17.4588 14.6305 17.6101 14.9707L18.9143 18.0046C19.8973 17.0595 20.6534 15.9631 21.1826 14.7155C21.7308 13.468 22.0049 12.1543 22.0049 10.7744ZM12.0244 20.7549C12.497 20.7549 12.9506 20.7265 13.3854 20.6698C13.839 20.5942 14.2832 20.4997 14.718 20.3863L11.7692 13.3829H7.09085V16.4168C7.09085 17.0027 7.21372 17.5509 7.45945 18.0613C7.70518 18.5716 8.03597 19.0253 8.45183 19.4223C8.86768 19.8192 9.35915 20.1405 9.92622 20.3863C10.4933 20.6131 11.0982 20.736 11.7409 20.7549H12.0244Z" fill="black"/>
+<svg width="25" height="24" viewBox="0 0 25 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
One alternate approach is to create an Icon component in the dls (if there's isn't one already). 

This icon component is theme aware, meaning that it passes the fill color based on the theme. 

You can set the fill color in scss using the following syntax: 

```
svg {
	path {
		fill: (color)
	}
}

```

Reply:
that works :D